src/components/Button.tsx

```diff
@@ -34,6 +34,7 @@ const ColouredButton = styled(ButtonBase)`
 `;
 
 const Outline = styled(ButtonBase)`
+	background: #fff;
```

Comment:
I would only fall back to `white` if the `bg` prop is not passed to it maybe? Not a strong opinion on this.

Reply:
I agree @sradevski , that makes sense